mithril-aggregator/benches/cardano_transactions.rs

```diff
@@ -0,0 +1,60 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use sqlite::ConnectionThreadSafe;
+use std::sync::Arc;
+
+use mithril_aggregator::{
+    database::repository::CardanoTransactionRepository, services::TransactionStore,
+};
+use mithril_common::{entities::CardanoTransaction, test_utils::TempDir};
+use mithril_persistence::sqlite::ConnectionBuilder;
+
+fn cardano_tx_db_connection() -> ConnectionThreadSafe {
+    let db_path =
+        TempDir::create("aggregator_benches", "bench_store_transactions").join("cardano_tx.db");
+
+    if db_path.exists() {
+        std::fs::remove_file(db_path.clone()).unwrap();
+    }
+
+    ConnectionBuilder::open_file(&db_path)
+        .with_migrations(
+            mithril_aggregator::database::cardano_transaction_migration::get_migrations(),
+        )
+        .build()
+        .unwrap()
+}
+
+fn generate_transactions(nb_transactions: usize) -> Vec<CardanoTransaction> {
+    (0..nb_transactions)
+        .map(|i| {
+            CardanoTransaction::new(
+                format!("tx_hash-{}", i),
+                i as u64,
+                i as u64 + 1,
+                format!("block_hash-{}", i),
+                i as u64 + 2,
+            )
+        })
+        .collect()
+}
+
+fn bench_store_transactions(c: &mut Criterion) {
+    const NB_CARDANO_TRANSACTIONS: usize = 100000;
```

Comment:
Maybe multiple values could be benchmarked up to at least 10/100 million records?

Also the value could be easier to read like this:
```suggestion
    const NB_CARDANO_TRANSACTIONS: usize = 100_000;
```

Reply:
Done from 10K -> 1M transactions, see results below.
Replaced as suggested:
`const NB_CARDANO_TRANSACTIONS: usize = 1_000_000;`